utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -299,10 +300,14 @@ def build(self, host_target):
                 'armv6 armv6m armv7 armv7m armv7em')
 
         llvm_enable_projects = ['clang']
+        llvm_enable_runtimes = []
 
         if self.args.build_compiler_rt and \
                 not self.is_cross_compile_target(host_target):
-            llvm_enable_projects.append('compiler-rt')
+            llvm_enable_runtimes.append('compiler-rt')
+            build_targets.append("runtimes")
+            # This is needed to build correctly TSan under Linux
+            llvm_cmake_options.define('COMPILER_RT_USE_BUILTINS_LIBRARY', 'ON')
```

Comment:
This may work for runtimes, but not for the sanitizers -- the best course of action would be to build them in a separate build product